packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -0,0 +1,60 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Error from '../Error.vue';
+import {
+    i18n, createStore
+} from './helpers/setup';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('Error', () => {
+    allure.feature('Checkout-Error');
+    const propsData = {};
+
+    it('should be defined', () => {
+        // Arrange & Act
+        const wrapper = shallowMount(Error, {
+            store: createStore(),
+            i18n,
+            localVue,
+            propsData
+        });
+
+        // Assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    it('should show the header', () => {
+        // Arrange & Act
+        const wrapper = shallowMount(Error, {
+            store: createStore(),
+            i18n,
+            localVue,
+            propsData
+        });
+
+        const header = wrapper.find('[data-test-id="checkout-error-page-header"]');
```

Comment:
Do we prefer this to importing the `Header` component and using `wrapper.findComponent(Header)`?

Reply:
I'm not sure I follow. The header in this context refers to the error page's heading (maybe I should rename it to `heading`), not the Checkout Header component.

Reply:
Renaming it to heading sounds good 👍